src/statespaceset.jl

```diff
@@ -262,3 +264,76 @@ function standardized_timeseries(d::AbstractStateSpaceSet)
     end
     return xs, means, stds
 end
+
+
+#####################################################################################
+#                          covariance/correlation matrix                            #
+#####################################################################################
+import Statistics: cov, cor
+using Statistics: mean, std
+using StaticArraysCore: MMatrix, MVector, SMatrix, SVector
+
+"""
+    cov(d::StateSpaceSet) → m::SMatrix
+
+Compute the covariance matrix `m` from the columns of `d`, where `m[i, j]` is the covariance
+between `d[:, i]` and `d[:, j]`.
+"""
+cov(x::AbstractStateSpaceSet) = fastcov(x.data)
+
+"""
+    mean_and_cov(d::StateSpaceSet) → μ, m::SMatrix
+
+Returns a tuple of the column means `μ` and covariance matrix `m`. 
+
+Column means are always computed for the covariance matrix, so this is faster 
+than computing both quantities separately.
+"""
+mean_and_cov(x::AbstractStateSpaceSet) = fastmean_and_cov(x.data)
+
+"""
+    cor(d::StateSpaceSet) → m::SMatrix
+
+Compute the corrlation matrix `m` from the columns of `d`, where `m[i, j]` is the 
+correlation between `d[:, i]` and `d[:, j]`.
+"""
+cor(x::AbstractStateSpaceSet) = fastcor(x.data)
```

Comment:
What's the difdference between correlation and covariance? AAren't they the same thing up to a normalization factor? We should say this.